x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/document_fields/fields/field_types/flattened_type.tsx

```diff
@@ -41,48 +42,57 @@ export const FlattenedType = React.memo(({ field }: Props) => {
   return (
     <>
       <EditFieldSection>
-        {/* depth_limit */}
-        <EditFieldFormRow
-          title={
-            <h3>
-              {i18n.translate('xpack.idxMgmt.mappingsEditor.depthLimitTitle', {
-                defaultMessage: 'Set depth limit',
-              })}
-            </h3>
-          }
-          description={i18n.translate('xpack.idxMgmt.mappingsEditor.depthLimitDescription', {
-            defaultMessage:
-              'The maximum allowed depth of the flattened object field, in terms of nested inner objects.',
-          })}
-        >
-          <UseField path="depth_limit" config={getFieldConfig('depth_limit')} component={Field} />
-        </EditFieldFormRow>
-
         {/* index */}
-        <IndexParameter />
+        <IndexParameter
+          config={getFieldConfig('index_options_flattened')}
+          indexOptions={PARAMETERS_OPTIONS.index_options_keyword}
```

Comment:
Should this be `PARAMETERS_OPTIONS.index_options_flattened`?

Reply:
Whoops! Good catch!